### DISCUSSION TITLE
Git Push SSH keys not working

### DISCUSSION THREAD

**Discussion by jlmghost** [community/community#22394] [New to GitHub]
<p>Hey Im new to coding and using github, so far ive set up and linked my ssh keys to my github account. And now im trying to push some local commits into git hub using “git push origin master” my machine asks for a username &amp; password. But when I enter the login it says, “Support for password authentication was removed on August 13, 2021. Please use a personal access token instead.” Im lost because I enabled and connected my SSH keys to my account but its not working. Can someone help? Thank you.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/b/0b16f79e359c55ff6b7e9881619c3842a0e9f878.png" data-download-href="/uploads/short-url/1A6stxydwgVymzX1lWc3fUY0Lfq.png?dl=1" title="Screen Shot 2021-11-28 at 12.43.03 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/94644590/181070739-a830c0c4-b5a9-46d7-9de8-991e0d4d791b.png" alt="Screen Shot 2021-11-28 at 12.43.03 PM" data-base62-sha1="1A6stxydwgVymzX1lWc3fUY0Lfq" width="690" height="357" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/b/0b16f79e359c55ff6b7e9881619c3842a0e9f878_2_690x357.png, https://aws1.discourse-cdn.com/github/optimized/3X/0/b/0b16f79e359c55ff6b7e9881619c3842a0e9f878_2_1035x535.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/0/b/0b16f79e359c55ff6b7e9881619c3842a0e9f878_2_1380x714.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/b/0b16f79e359c55ff6b7e9881619c3842a0e9f878_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screen Shot 2021-11-28 at 12.43.03 PM</span><span class="informations">1786×926 238 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by airtower-luna** [comment, marked as answer]
<p>The repository URL in those logs is still the HTTPS one. To use SSH you have to replace it with the matching SSH URL using <a href="https://git-scm.com/docs/git-remote#Documentation/git-remote.txt-emset-urlem"><code>git remote set-url</code></a>:</p>
<pre><code class="lang-auto">git remote set-url origin git@github.com:user/repo.git
</code></pre>
<p>Replace <code>user</code> and <code>repo</code> as appropriate, you can find the full URL on your repository site (green “Code” button, select “SSH” in the popup).</p>

**Comment by jlmghost** [comment]
<p>Ok thank you, now after doing that I try to “git push origin master” and terminal replies</p>
<p>fatal: remote error:<br>
is not a valid repository name</p>

**Comment by airtower-luna** [comment]
<p>That sounds like something might still be wrong with the URL. You can check the configured URLs with:</p>
<pre><code class="lang-auto">git remote -v
</code></pre>
